src/litserve/api.py

```diff
@@ -3,6 +3,21 @@
 from abc import ABC, abstractmethod
 
 
+def no_batch_unbatch_message(obj):
+    return f"""
+        You set `max_batch_size > 1` but didn't implement batch() and unbatch().
```

Comment:
We are providing a default batch and unbatch implementation for tensors and ndarrays, so we could improve the messaging to be more specific.

```
You set `max_batch_size > 1`, but the default implementation for batch() and unbatch() only supports PyTorch tensors or NumPy ndarrays, while we found {TODO ADD TYPE HERE}.
Please implement these two methods in {obj.__class__.__name__}.
```

Reply:
updated to this ^